test/scripts/generateJson.ts

```
@@ -67,18 +67,75 @@ function getErrorResponse(outputResponse?: responseType) {
+
+  const { body } = testCase.input.request;
+  const bodyElement = body.length === 1 ? body[0] : body;
+
+  const queryParams = (function () {
+    switch (version) {
+      case 'v0':
+        return getQueryParameters(bodyElement, rawParams);
+      case 'v1':
+        return getQueryParameters(bodyElement.event, rawParams);
+      case 'v2':
+        const { query_parameters } = bodyElement.request;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)